File: integration_tests/java/gamut/DiscriminatorBaz.java
package com.jsontypedef.jtdcodegendemo;




public class DiscriminatorBaz extends Discriminator {

  
  @JsonProperty("bazThing")
  private Object bazThing;


  
  public DiscriminatorBaz() {
  }
  


  public Object getBazThing() {
    return bazThing;
  }

  public void setBazThing(Object bazThing) {
    this.bazThing = bazThing;
  }

}

File: integration_tests/csharp/gamut/DiscriminatorBaz.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Newtonsoft.Json.Linq;

namespace Jtd.JtdCodegenDemo
{
    
    public class DiscriminatorBaz : Discriminator
    {

        [JsonProperty("foo")]
        public const string Foo = "baz";


        [JsonProperty("bazThing")]
        public object BazThing { get; set; }





    }
}
